haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
What if we logged a warning if the value was over a certain (much more conservative) value? There's a pretty wide range of unusably long delays between the default (2 seconds) and an hour -- this would give us the ability to be more aggressive with the warning, while not preventing people from debugging. Otherwise, we have to put the upper bound well outside the range of "this might be usable"

I don't have terrbily strong feelings on the matter. If we do put in an upper bound like 3600, I'd like the docs to have a parenthetical `(1 hour)`, just so the reader can see it at a glance. But other than that, I'll go along with whatever you guys decide.